src/OpenSage.Game/Data/W3d/W3dRgbF.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.IO;
+using System.Runtime.InteropServices;
+
+namespace OpenSage.Data.W3d
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct W3dRgbF
+    {
+        public float R { get; private set; }
```

Comment:
We're actually in the process of removing W3D-specific colour types, so please could you use `ColorRgbF` instead?